.github/labeler.yml

```diff
@@ -0,0 +1,255 @@
+# Package labeling rules - only labels that exist on GitHub
```

Comment:
Is there any way to automate this list? Looking quickly, it seems that the part of the label after `[Package] ` can just be coerced into lowercase and spaces replaced with hyphens to construct the directory path. For example, `[Package] API Fetch` -> `packages/api-fetc/**`. The only one where this is not the case is the `[Package] E2E Tests`.

We just removed a manually-curated list in #72672, so want to make sure that adding a new file for something similar is the desired solution.

I don't know if this is possible, but what if there was a step prior to this one that queried for all of the active `[Package]` labels for the repository and then created this file or a list on the fly each time?

Reply:
Actually:
- I've fixed the two labels that had a typo/mismatch
- For the exceptions, we can use the label description as a backup. They contain the patch of the package they should be "assigned" to, so we can use label descriptions to do the match in the workflow.